.gitignore

```diff
@@ -3,5 +3,5 @@
 /ooniprobe.xcodeproj/xcuserdata
 /ooniprobe.xcworkspace/xcuserdata/
 /Pods/
-
+Podfile.lock
```

Comment:
I don't think this is correct. The `Podfile.lock` should be committed. See for example what it says here: https://github.com/thoughtbot/liftoff/issues/30